build.properties

```diff
@@ -88,9 +88,9 @@ jfx.gradle.version=5.3
 jfx.gradle.version.min=4.8
 
 # Toolchains
-jfx.build.linux.gcc.version=gcc7.3.0-OEL6.4+1.1
-jfx.build.windows.msvc.version=VS2017-15.5.5+1.1
-jfx.build.macosx.xcode.version=Xcode9.4-MacOSX10.13+1.0
+jfx.build.linux.gcc.version=gcc8.2.0-OL6.4+1.0
+jfx.build.windows.msvc.version=VS2017-15.9.6+1.0
+jfx.build.macosx.xcode.version=Xcode10.1-MacOSX10.14+1.0
```

Comment:
since Xcode10.2-MacOSX10.14+4.0 fail, should we also define a maximum version? Or is it assumed that this exact version needs to be used?

Reply:
This is intended to be the exact version of Xcode and the SDK used by the build (independent of the version of macOS running on the build machine).